pyproject.toml

```diff
@@ -5,7 +5,7 @@
 package-mode = false
 
 [tool.poetry.dependencies]
-python = "^3.8.10"
+python = "^3.10"
```

Comment:
K8s charm is only running on jammy, so bump the version to be able to update coverage: https://github.com/canonical/pgbouncer-k8s-operator/pull/440

Reply:
It the lib/code sharing will not be the problem: ACK.

Reply:
This is a charm specific dependency, so it shouldn't affect the lib. I think all the linting so far should be 3.8 compatible.